src/intentManager.js

```diff
@@ -219,70 +154,38 @@ define([
         icon: 'icon-vellum-android-intent',
         isODKOnly: true,
         isTypeChangeable: false,
-        intentTag: null,
         init: function (mug, form) {
+            mug.p.intentXmlns = mug.p.intentXmlns || DEFAULT_XMLNS;
         },
         spec: {
             androidIntentAppId: {
                 lstring: 'Intent ID',
                 visibility: 'visible',
                 widget: androidIntentAppId,
-                serialize: function (value, key, mug, data) {
```

Comment:
I thought that the excel workflow was specifically unsupported?

I suppose I could change this to accept both forms on paste, but copy will always use the new form, and then change it to the simple on in a couple of weeks. Thoughts?